tests/compat.sh

```diff
@@ -799,16 +798,13 @@ wait_client_done() {
     echo "EXIT: $EXIT" >> $CLI_OUT
 }
 
-# run_client <name> <cipher>
+# run_client PROGRAM_NAME STANDARD_CIPHER_SUITE PROGRAM_CIPHER_SUITE
 run_client() {
     # announce what we're going to do
     TESTS=$(( $TESTS + 1 ))
-    VERIF=$(echo $VERIFY | tr '[:upper:]' '[:lower:]')
-    TITLE="`echo $1 | head -c1`->`echo $SERVER_NAME | head -c1`"
+    TITLE="${1%"${1#?}"}->${SERVER_NAME%"${SERVER_NAME#?}"}"
     TITLE="$TITLE $MODE,$VERIF $2"
-    printf "%s " "$TITLE"
-    LEN=$(( 72 - `echo "$TITLE" | wc -c` ))
-    for i in `seq 1 $LEN`; do printf '.'; done; printf ' '
+    printf "%s %.*s " "$TITLE" "$((72 - ${#TITLE}))" ........................................................................
```

Comment:
```suggestion
    DOTS72="........................................................................"
    printf "%s %.*s " "$TITLE" "$((72 - ${#TITLE}))" "$DOTS72"
```
Minor: for a good readability.